engine/orchestration/src/main/java/org/apache/cloudstack/engine/orchestration/NetworkOrchestrator.java

```diff
@@ -578,7 +583,17 @@ public void doInTransactionWithoutResult(final TransactionStatus status) {
 
                 _networkOfferingDao.persistDefaultL2NetworkOfferings();
 
-                _kvmMtuSize = NumbersUtil.parseInt(_configDao.getValue(KvmMtuSize.key()), 0);
+                final List<ClusterVO> clusterVOs = _clusterDao.listByHypervisorAndCluster(HypervisorType.KVM, Cluster.ClusterType.CloudManaged);
+
+                if ((clusterVOs != null) && !clusterVOs.isEmpty()) {
```

Comment:
Can you please update this if conditional to `if (!CollectionUtils.isEmpty(clusterVOs)) { ... }`?
The "isEmpty" is a Null-safe check; thus, it is a great option for checking if a collection is empty (returning true if null or empty).